xml/System.Net.Security/SslClientAuthenticationOptions.xml

```diff
@@ -193,7 +193,7 @@
         <ReturnType>System.Security.Cryptography.X509Certificates.X509ChainPolicy</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets an optional customized policy for remote certificate validation. If not <see langword="null" />, <see cref="P:System.Net.Security.SslClientAuthenticationOptions.CertificateRevocationCheckMode" /> and <see cref="T:System.Net.Security.SslCertificateTrust" /> are ignored.</summary>
+        <summary>Gets or sets an optional customized policy for remote certificate validation. If not <see langword="null" />, <see cref="P:System.Net.Security.SslClientAuthenticationOptions.CertificateRevocationCheckMode" /> and <see cref="T:System.Net.Security.SslCertificateTrust" /> used to create the <see cref="P:System.Net.Security.SslClientAuthenticationOptions.ClientCertificateContext" /> instance are ignored.</summary>
```

Comment:
That does not seem quite right. It seems like we still use `SslCertificateTrust.sendTrustInHandshake` regardless. e.g. presence of the policy has only impact on remote certificate validation.

Reply:
Further, it looks like if sendTrustInHandshake is true, the SslCertificateTrust's list is what it sends.  Of course, that's only relevant for the server side, for the client side it is true that the SslCertificateTrust is ignored.

Usually the goal of a summary is just one sentence, and perhaps the problem here is that what happens when it's not null is complicated.

I'd delete the whole second sentence, and move it to remarks:

```
When the value of this property is not null, the value of the CertificateRevocationCheckMode property is ignored.

When the value of this property is not null, any SslCertificateTrust associated with the ClientCertificateContext is ignored.
```

And for the server side

```
When the value of this property is not null, the value of the CertificateRevocationCheckMode property is ignored.

When the value of this property is not null and a SslCertificateTrust value was provided to the value in the ServerCertificateContext property, the trust list SslCertificateTrust will be sent in the handshake (when so configured) but the trust settings from this property will be used when verifying any client authentication certificates.
```

Reply:
I assume the last sentence on the server side version should be

```diff
- but the trust settings from this property will be used when verifying any client authentication certificates.
+ but the trust settings from this property will not be used when verifying any client authentication certificates.
```

but otherwise great suggestion.